src/account/ModuleManagerInternals.sol

```diff
@@ -227,6 +229,16 @@ abstract contract ModuleManagerInternals is IModuleManager {
         }
     }
 
+    function _onInstall(address module, bytes calldata data, bytes4 interfaceId) internal {
```

Comment:
Should we just remove the unused `_onInstall` above this one?

Reply:
yep, done!